rfcs/0175-appeals-council.md

```diff
@@ -0,0 +1,170 @@
+
+We contend that this ambiguity has allowed the moderation team to shape community values without consent, either direct or implied. The exercise of power in this manner, to suppress reasonable dissent and foster an illusion of unity, has a chilling effect that hinders productive dialogue and ultimately harms the project.
+
+This proposal seeks to establish clear foundations for a shared understanding of moderation practices through the *Memorandum of Understanding on Equitable Moderation* and to ensure these foundations are upheld in practice. It aims to prevent the NixOS foundation mission statement from being misused to enforce a homogenizing moderation policy.
+
+The *Memorandum* should be seen as a normatively binding interpretation of the moderation team's mission statement.
+
+This document aims to articulate an overlapping consensus on reasonable expectations for anyone participating as an equal member in this community, emphasizing that community members are accountable to moderation and vice versa.
+
+# Detailed Design
+[design]: #detailed-design
+
+## Goals
+
+- Adopt the *Memorandum of Understanding on Equitable Moderation* to foster a shared understanding of equitable moderation practices that accommodate a community with diverse sociopolitical views.
+- Ensure transparent accountability of moderation practices to community members.
+- Ground community member accountability in publicly available general principles, providing a stable basis for behavioral expectations.
+- Make wide-ranging decisions regarding community member exclusion understandable.
+- Promote a diverse moderation team to balance diverse perspectives on controversial issues.
+- Establish an *Appeals Council* to hold moderation team members and procedures accountable and to allow for the revision of significant decisions if they fail to meet the community's reasonable standards.
+
+## Memorandum of Understanding on Equitable Moderation
+
+In response to the impression left by past events and with a fervent desire for improvement, we adopt the following principles and maxims as the *Memorandum of Understanding on Equitable Moderation*. These principles serve to clarify the overlapping consensus on reasonable expectations for community participation:
+
+1) We, as moderators, commit to basing decisions on general principles and concrete reasons that apply equally to all community members, irrespective of characteristics such as sociopolitical orientation, economic status, identity, or religion.
+2) We adhere to the principles of *equality* and *objectivity*. Equality refers to the status of each community member, while objectivity pertains to the consistent, unbiased reasoning behind our decisions.
+3) A *concrete reason* is a verifiable state of affairs that can be demonstrated to other community members upon request.
```

Comment:
This is an impossibly high standard for moderation.

Reply:
How so? Could you elaborate?

Reply:
> How so? Could you elaborate?

Consider this passage:

> A *concrete reason* is a verifiable state of affairs that can be demonstrated to other community members upon request.

How do you formally verify and/or demonstrate that someone is trolling? A taped confession?

Reply:
I don't see why it wouldn't be possible to do so in many cases. In fact, if the moderation cannot present evidence enough to base a decision on I don't think it's a good baseline at all. It means that if, for example, I am a moderator the moderation could be totally different than if another person is, and the rest of the community has no way to reliably know.

Reply:
> How do you formally verify and/or demonstrate that someone is trolling? A taped confession?

Is this an argument for or against trolling?

Reply:
I don't think its either?  Its saying that calling someone a troll is an easy way to dismiss them, even if they're expressing their genuine beliefs.  Now, their genuine beliefs may not be something that's allowed to be expressed on Discourse, so say that.

Reply:
I think most of the coder's covenant is easy to demonstrate reasoning. Just linking to the personal attack, trolling, etc should be enough context for everyone to agree.

Maybe we shouldn't be shifting goal posts on what moderation team is supposed to do?